docs/user/tutorial/70-page-management.rst

```diff
@@ -4,7 +4,10 @@ Managing pages
 
 Our menu of pages is growing.
 
-.. todo:: screenshots of menu
+|page_tree|
```

Comment:
the thing is, if I want to align the image to the left, it floats and the text that is below, will move up and stick to the right edge of the image.
Using the above method will keep the space on the right side empty.